docs/mkdocs.yml

```diff
@@ -170,6 +170,7 @@ nav:
           - Visualize your graph: how-tos/visualization.ipynb
           - Add runtime configuration: how-tos/configuration.ipynb
           - Add node retries: how-tos/node-retries.ipynb
+          - How to force function calling agent to structure output: how-tos/react-agent-structured-output.ipynb
```

Comment:
```suggestion
          - Force function calling agent to structure output: how-tos/react-agent-structured-output.ipynb
```

Reply:
(nit) removed "how to" from this. also, let's make sure the title is the same here and in the notebook itself